### ISSUE TITLE
Zombie mailing list

### ISSUE THREAD

**Issue by brucelilly** [att/ast#992]
The URL at the end of the monthly reminder, viz.
http://lists.research.att.com/mailman/options/<specific details elided>
doesn't work; it results in an error page.

Mail about this to mailman-owner@lists.research.att.com bounces:

There was a temporary problem delivering your message to mailman-owner@lists.research.att.com. Gmail will retry for 47 more hours. You'll be notified if the delivery fails permanently.

The response was:

The recipient server did not accept our requests to connect.

It appears that mail to mailman-request@lists.research.att.com is also disappearing into a black hole in the death star :-), so there's no way to unsubscribe to what is essentially a dead list (no activity on ast-developers in 7 months other than the monthly reminders).

**Comment by krader1961** [comment]
Yeah, the lists were inoperative for a while a couple of years ago around the time the project was moved to Github. We should update the README.md file to mention the lists solely as historical artifacts.

**Comment by krader1961** [comment]
The test message I sent resulted in an undeliverable message.

**Comment by brucelilly** [comment]
I'm not surprised.  If there's somebody at AT&T reading this, it would be appreciated if either the mailman-request@lists.research.att.com address could be made functional again (so that people can unsubscribe), or failing that just nuke the entire list(s) (including the monthly reminder emails).
